docs/manual/gettingStarted/helpAndTroubleshooting/knownIssues.md

```diff
@@ -112,48 +112,60 @@ Syncing Playnite's library files via cloud solutions (OneDrive, Google Drive) ca
+2. Consider updating your Intel drivers to fix this problem. Alternatively, you can disable hardware acceleration in Playnite, although this is not recommended as that will cause general performance issues in Playnite.
 
 ### AuthorizationManager check failed error at Startup
 
-If you are using portable version of Playnite then Windows may be blocking loading of some script files. Go to Playnites program folder, under `Scripts` subfolder locate script files reporting AuthorizationManager error, open file properties (via right-click on the file) and unblock the files.
+If you are using the portable version of Playnite and encounter the AuthorizationManager error at startup, it may be due to Windows blocking the loading of certain script files. To resolve this, follow these steps:
+
+1. Navigate to Playnite's program folder.
+2. Within the folder, locate the `Scripts` subfolder.
+3. Identify the script files that are reporting the AuthorizationManager error.
+4. Right-click on each of these files and open their properties.
+5. Unblock the files to allow them to run without issue.
 
 ### Extensions menu is grayed Out / PowerShell extensions not loading
 
 On Windows 7 and 8, [installing PowerShell 5](https://www.microsoft.com/en-us/download/details.aspx?id=54616) resolves this issue.
 
 ### Settings being Reset or general Data Loss
 
-Users reported these issues when installing Playnite via Chocolatey. Official installation methods are recommended to avoid these problems.
+Users have reported encountering these issues when installing Playnite via Chocolatey. To minimize the risk of these problems, it is recommended to use the official installation methods.
 
 ### WebP Images don't display properly
 
-Install the [WebP Image Extensions](https://www.microsoft.com/en-us/p/webp-image-extensions/9pg2dk419drg) from Microsoft Store.
+Install the [WebP Image Extensions](https://www.microsoft.com/en-us/p/webp-image-extensions/9pg2dk419drg) from the Microsoft Store.
 
 ### Rendering issues while streaming Playnite with Sunshine/Moonlight/Parsec
 
 Incompatibility between streaming software and Playnite's UI library (WPF) can cause rendering problems. Report these issues to the developers of the streaming software for potential fixes.
 
 ### Playnite Takes Longer to Start After Windows Update
 
-Open Command Prompt as admin and these run commands:
-```
-c:\Windows\Microsoft.NET\Framework\v4.0.30319\ngen.exe update
-c:\Windows\Microsoft.NET\Framework64\v4.0.30319\ngen.exe update
-```
+If you've noticed that Playnite takes longer to start after a Windows update, you can try the following steps:
+
+1. Open the Command Prompt as an administrator.
+2. Run the following commands:
+
+    ```
+    c:\Windows\Microsoft.NET\Framework\v4.0.30319\ngen.exe update
+    c:\Windows\Microsoft.NET\Framework64\v4.0.30319\ngen.exe update
+    ```
+
+These commands may help to improve Playnite's performance.
```

Comment:
This is not true. ngen only affects startup time by pre-compiling .NET libraries meaning that they don't have to go through JIT compilation at runtime. It doesn't affect performance in any other way.

Reply:
Fixed https://github.com/JosefNemec/PlayniteDocs/pull/9/commits/4242384f52d24b7639a6893869a8a6dc1aab3b88

Reply:
That sentence is now completely redundant, the entire paragraph is about issue of Playnite being slow to start.

Reply:
Removed in https://github.com/JosefNemec/PlayniteDocs/pull/9/commits/922cf250cf7e942064994f493cf866fd9335597d